app/services/reports/monthly_bulk_report_service.rb

```diff
@@ -41,5 +38,25 @@ def file_name
     def bulk_report
       MonthlyBulkSerializer.new(@first_day_of_the_month).to_csv
     end
+
+    def load_file_to_aws_bucket
+      result = nil
+
+      3.times do
+        result = bucket.load(temp_file)
+
+        break if result.successful?
+      end
+
+      raise(result.error) unless result.successful?
+    end
+
+    def bucket
+      @_bucket ||= DefraRuby::Aws.get_bucket(bucket_name)
+    end
+
+    def bucket_name
+      ENV["AWS_BULK_EXPORT_BUCKET"]
```

Comment:
We have another one of those unwritten conventions 😳 

Excluding initializers, for any config the app needs we tend to load it once in `config/application.rb` and then refer to `Rails.configuration` within the rest of the app.

I'll be honest this just always seems to have been the case, so I can't say why 😕 but as we seem to have a convention we should probably stick to it for now.

Reply:
Will do :)